### PR TITLE
patch: clarifying offline vs inactive

### PR DESCRIPTION
**Pull request by rmorillo24**
Modified the section Inactive vs. Offline to make it more clear how it works, when it's billed and when the billing is resumed.

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/manage/billing.md [hunk 1]
```diff
@@ -71,7 +71,11 @@ Marking a device as `Inactive` is a [device action][device-action]. Device actio
...
 
-If you have a device offline but active you **will still be billed for that device.** Those devices are still deployed in the field and ready to be used at any time, if you make the decision to have your devices offline intentionally, you will still have full visibility to remotely monitor the device and are able to update it. You will not be able to monitor inactive devices and won't be charged for them after the de-activation fee until you reactive them on balenaCloud.
+- Inactive: The device has been deactivated, or has been preregistered but has not yet connected to the balenaCloud API.
+- Offline: The device is not connected to cloudlink and has not any recent API communications.
+
+If you have a device offline _and_ active you **will still be billed for that device.** Those devices are usually still deployed in the field and ready to be used at any time. If you make the decision to have your devices offline intentionally, you will still see them in your fleet and will have full visibility to remotely monitor the devices or to update them, actions that will take effect as soon as they become online.
```
Comment (klutchell) [inline pages/learn/manage/billing.md:79]: > you will still see them in your fleet and will have full visibility to remotely monitor the devices or to update them, actions that will take effect as soon as they become online.

This seems a but confusing, maybe something like this instead?
```suggestion
If you have a device offline _and_ active you **will still be billed for that device.** Those devices are usually still deployed in the field and ready to be used at any time. If you make the decision to have your devices offline intentionally, you will still see them in your fleet and and can take some actions, such as applying updates, that will take effect as soon as they come online.
```

File: pages/learn/manage/billing.md [hunk 2]
```diff
@@ -71,7 +71,11 @@ Marking a device as `Inactive` is a [device action][device-action]. Device actio
...
+
+Once you deactivate a device, you will not be able to monitor nor to apply updates to it, and you won't be charged after the de-activation fee until they become online again.
```
Comment (klutchell) [inline pages/learn/manage/billing.md:81]:
```suggestion
Once you deactivate a device, you will not be able to monitor nor apply updates to it, and after the de-activation fee you won't be charged again until they come online.
```

**Comment by rmorillo24** [comment]
Thank you Kyle. Much better with your suggestions

**Comment by klutchell** [comment]
@rmorillo24 can you squash 4 commits into 1?
https://stackoverflow.com/a/5201642

File: pages/learn/manage/billing.md [hunk 3]
```diff
@@ -71,7 +71,11 @@ Marking a device as `Inactive` is a [device action][device-action]. Device actio
 
 ## Inactive vs Offline Devices
 
-An inactive device is: The device has been deactivated or has been preregistered but has not yet connected to the balenaCloud API.
-An offline device is: The device is offline and is not connected to cloudlink and has not any recent API communications.
+To understand the difference between Inactive and Offline, we define them the following way:
```
Comment (vipulgupta2048) [inline pages/learn/manage/billing.md:74]:
```suggestion
To understand the difference between inactive and offline devices, we define them in the following ways:
```

File: pages/learn/manage/billing.md [hunk 4]
```diff
@@ -71,7 +71,11 @@ Marking a device as `Inactive` is a [device action][device-action]. Device actio
...
 
-If you have a device offline but active you **will still be billed for that device.** Those devices are still deployed in the field and ready to be used at any time, if you make the decision to have your devices offline intentionally, you will still have full visibility to remotely monitor the device and are able to update it. You will not be able to monitor inactive devices and won't be charged for them after the de-activation fee until you reactive them on balenaCloud.
+- Inactive: The device has been deactivated, or has been preregistered but has not yet connected to the balenaCloud API.
```
Comment (vipulgupta2048) [inline pages/learn/manage/billing.md:76]:
```suggestion
- **Inactive**: The device has been either [deactivated](https://www.balena.io/docs/learn/manage/actions/#deactivate-device) or [preregistered](https://www.balena.io/docs/learn/more/masterclasses/advanced-cli/#52-preregistering-a-device) but hasn't connected to the balenaCloud API yet.
```

File: pages/learn/manage/billing.md [hunk 5]
```diff
@@ -71,7 +71,11 @@ Marking a device as `Inactive` is a [device action][device-action]. Device actio
...
+- Offline: The device is not connected to cloudlink and has not had any recent API communications.
```
Comment (vipulgupta2048) [inline pages/learn/manage/billing.md:77]:
```suggestion
- **Offline**: The device is not connected to cloudlink and didn't have any recent API communications.
```

File: pages/learn/manage/billing.md [hunk 6]
```diff
@@ -71,7 +71,11 @@ Marking a device as `Inactive` is a [device action][device-action]. Device actio
...
+
+If you have a device offline _and_ active you **will still be billed for that device.** Those devices are usually still deployed in the field and ready to be used at any time. If you make the decision to have your devices offline intentionally, you will still see them in your fleet and and can take some actions, such as applying updates, that will take effect as soon as they come online.
```
Comment (vipulgupta2048) [inline pages/learn/manage/billing.md:79]:
```suggestion
If you have a device **offline and active**, you will **still be billed for that device**. Those devices are usually still deployed in the field and ready to be used at any time. If you make the decision to have your devices offline intentionally, you will still be able see them in your fleet and can take some actions such as applying updates, that will take effect as soon as they come online.
```

File: pages/learn/manage/billing.md [hunk 7]
```diff
@@ -71,7 +71,11 @@ Marking a device as `Inactive` is a [device action][device-action]. Device actio
...
+
+Once you deactivate a device, you will not be able to monitor nor apply updates to it, and after the de-activation fee you won't be charged again until they come online.
```
Comment (vipulgupta2048) [inline pages/learn/manage/billing.md:81]:
```suggestion
Once you deactivate a device, you will not be able to monitor nor apply updates to that device. After the de-activation fee is charged, you won't be charged again until the device comes back online.
```

docs always explicit

**Review by vipulgupta2048 — COMMENTED** [review]
Good one @rmorillo24 Added some markdown changes as well for better presentation.

**Review by vipulgupta2048 — APPROVED** [review]

**Review by rahul-thakoor — APPROVED** [review]

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Pull request merged**